utils/githooks/pre-commit.d/30-Jenkinsfile.sh

```diff
@@ -25,6 +25,7 @@ CURL_VERBOSE=${CURL_VERBOSE:-""}
 CURL_PROXY="${CURL_PROXY:+-x }${CURL_PROXY:-}"
 CURL_OPTS="$CURL_PROXY $CURL_VERBOSE -s"
 URL="https://$HOST/pipeline-model-converter/validate"
+# shellcheck disable=SC2086
 if ! output=$(curl $CURL_OPTS -s -X POST -F "jenkinsfile=<${1:-Jenkinsfile}" "$URL"); then
```

Comment:
SC2086 is usually pretty easy to resolve:

```suggestion
CURL_OPTS=($CURL_PROXY $CURL_VERBOSE -s)
URL="https://$HOST/pipeline-model-converter/validate"
if ! output=$(curl "${CURL_OPTS[@]}" -s -X POST -F "jenkinsfile=<${1:-Jenkinsfile}" "$URL"); then
```

Clearly I should have done that in the first place.

Reply:
Actually, no. It doesn't like that either
https://github.com/daos-stack/daos/actions/runs/12164183236/job/33925207435?pr=15552

Reply:
Pushed with more quotes